server/session/security_cookie.ts

```diff
@@ -69,6 +69,6 @@ export function getSecurityCookieOptions(
       return { isValid: true, path: '/' };
     },
     isSecure: config.cookie.secure,
-    sameSite: config.cookie.isSameSite || 'None',
+    sameSite: config.cookie.isSameSite || undefined,
```

Comment:
In this case `false` will become `undefined` and `'None'` will is still `'None'`, is it exptected?

Reply:
yes, this is expected